libtransmission/announcer.h

```diff
@@ -70,10 +70,7 @@ using tr_tracker_callback = std::function<void(tr_torrent&, tr_tracker_event con
 class tr_announcer
 {
 public:
-    [[nodiscard]] static std::unique_ptr<tr_announcer> create(
-        tr_session* session,
-        tr_announcer_udp&,
-        std::atomic<size_t>& n_pending_stops);
+    [[nodiscard]] static std::unique_ptr<tr_announcer> create(tr_session* session, tr_announcer_udp&);
```

Comment:
With `n_pending_stops` removed, maybe `announcer.h` doesn't need to `#include <atomic>` anymore?

Reply:
81fcef9fafde094bc06c09c918d3346c318d46b6